src/bridge/command.rs

```diff
@@ -48,26 +48,26 @@ fn build_nvim_cmd() -> TokioCommand {
 }
 
 // Creates a shell command if needed on this platform (wsl or macos)
-fn create_platform_shell_command(command: String) -> Option<StdCommand> {
+fn create_platform_shell_command(command: &str, args: Vec<&str>) -> Option<StdCommand> {
     if cfg!(target_os = "windows") && SETTINGS.get::<CmdLineSettings>().wsl {
         let mut result = StdCommand::new("wsl");
-        result.args(&["$SHELL", "-lic"]);
-        result.arg(command);
+        result.args(&["$SHELL", "-lc"]);
+        result.arg(format!("{} {}", command, args.join(" ")));
 
         Some(result)
     } else if cfg!(target_os = "macos") {
-        let shell = env::var("SHELL").unwrap();
-        let mut result = StdCommand::new(shell);
-        result.args(&["-lic"]);
-        result.arg(command);
+        let mut result = StdCommand::new(command);
+        result.args(&args);
+
         Some(result)
     } else {
         None
     }
 }
 
 fn platform_exists(bin: &str) -> bool {
-    if let Some(mut exists_command) = create_platform_shell_command(format!("exists -x {}", bin)) {
+    // if let Some(mut exists_command) = create_platform_shell_command(format!("exists -x {}", bin)) {
```

Comment:
Even smaller nit: Delete the line rather than commenting it out. Git takes care of old things already, you don't need to preserve them manually.

Reply:
Oh! This is my mistake. I forgot to delete 😅  thank you!